backends/advanced/scripts/create_plugin.py

```
@@ -0,0 +1,431 @@
+    class_name = snake_to_pascal(plugin_name) + 'Plugin'
+
+    # Get plugins directory
+    script_dir = Path(__file__).parent
+    backend_dir = script_dir.parent
+    plugins_dir = backend_dir / 'src' / 'advanced_omi_backend' / 'plugins'
+    plugin_dir = plugins_dir / plugin_name
+
+    # Check if plugin already exists
+    if plugin_dir.exists() and not force:
+        print(f"❌ Error: Plugin '{plugin_name}' already exists at {plugin_dir}")
+        print(f"   Use --force to overwrite")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)